Serialiser_Engine/Convert/Bson.cs

```diff
@@ -177,6 +177,7 @@ private static void RegisterSerializers()
                 BsonSerializer.RegisterSerializer(typeof(object), new BH_ObjectSerializer());
                 BsonSerializer.RegisterSerializer(typeof(System.Drawing.Color), new ColourSerializer());
                 BsonSerializer.RegisterSerializer(typeof(MethodBase), new MethodBaseSerializer());
+                BsonSerializer.RegisterSerializer(typeof(MethodInfo), new MethodBaseSerializer());
```

Comment:
My comment here did not make it to the review for whatever reason.

Had commented here that I think we should initialising this the same way as the TypeSerialiser is being registred. That is, create the `MethodBaseSerializer` once, and then assign it to both MethodBase and MethodInfo, rather than creating two separate isntances of the `MethodBaseSerializer` class.

Reply:
Done 👍